apps/dashboard/app/controllers/files_controller.rb

```diff
@@ -229,21 +248,17 @@ def posix_file?
     @path.is_a?(PosixFile)
   end
 
-  def download?
-    params[:download]
-  end
```

Comment:
Can we keep this download? question and just use `fs_params[:download]`? (maybe even `fs_params[:download] == 'true'`)